tests/test_cli/test_scaffold/test_connection.py

```diff
@@ -91,6 +93,21 @@ def test_exit_code_equal_to_0(self):
         """Test that the exit code is equal to 0."""
         assert self.result.exit_code == 0
 
+    def test_copyright_header_year_is_current(self):
+        """Test that the year in copyright header is updated to current"""
+
+        incorrect_files, files_left_to_inspect = [], 2
+        current_year = datetime.now().year
+        path_pattern = f"**/{self.resource_name}/*.py"
+        for file in (Path(self.t) / self.agent_name).rglob(path_pattern):
+            content = file.read_text()
+            years_in_header = re.findall(r"Copyright.*([0-9]{4})", content)
+            if years_in_header != [str(current_year)]:
+                incorrect_files.append(file)
+            files_left_to_inspect -= 1
+        assert not "\n".join(map(str, incorrect_files))
+        assert not files_left_to_inspect
+
```

Comment:
very simple tests. The `files_left_to_inspect` counter is there to ensure that the scaffolded files are in fact inspected.